ingest/usecase/plugins/orderbook/claimbot/order.go

```diff
@@ -0,0 +1,183 @@
+package claimbot
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/osmosis-labs/osmosis/osmomath"
+	"github.com/osmosis-labs/sqs/domain"
+	"github.com/osmosis-labs/sqs/domain/mvc"
+	orderbookdomain "github.com/osmosis-labs/sqs/domain/orderbook"
+	orderbookgrpcclientdomain "github.com/osmosis-labs/sqs/domain/orderbook/grpcclient"
+	"github.com/osmosis-labs/sqs/log"
+
+	"go.uber.org/zap"
+)
+
+type order struct {
+	Orderbook domain.CanonicalOrderBooksResult
+	Orders    orderbookdomain.Orders
+	Err       error
+}
+
+// processOrderbooksAndGetClaimableOrders processes a list of orderbooks and returns claimable orders for each.
+func processOrderbooksAndGetClaimableOrders(
```

Comment:
I'm seeing only this function being covered by tests but there are many downstream function of varying branching complexity.

Could we spend some time covering those with tests?

I think it would pay off in the long-term as it would make any possible refactors drastically easier under simpler test coverage

Reply:
Added a several tests in the use case: https://github.com/osmosis-labs/sqs/pull/524/files#diff-5fa55bd6410efa835405589edaffbb41ba95f810344564647d2d711e8bc15b67